app/atoms/file.ts

```diff
@@ -1,8 +1,12 @@
 import type { ChangeEvent } from "react";
 import { atom } from "jotai";
-import { verifyAccept } from "~/utils";
+import { formatBytes, verifyAccept } from "~/utils";
 
 export const fileErrorAtom = atom<string | undefined>(undefined);
+fileErrorAtom.onMount = (setAtom) => {
+  setAtom("");
+};
+export const MAX_SIZE = 4_000_000; // 4MB
```

Comment:
I think we need a bit more size for the PDF's. Lets say 8MB.

Reply:
Resolved by [d18e0fc](https://github.com/Shelf-nu/shelf.nu/pull/719/commits/d18e0fc911abaff21fd630a074fa1829327726be)